src/ServiceInsight/App.xaml.cs

```diff
@@ -24,7 +26,14 @@ public App()
         protected override void OnStartup(StartupEventArgs e)
         {
             LogTo.Information("Starting the application...");
-            DXSplashScreen.Show(o => AboutView.AsSplashScreen(), null, null, null);
+
+            var doNotShowSplashScreen = e.Args.Length > 0 && e.Args.Contains ("--silent", StringComparer.OrdinalIgnoreCase);
```

Comment:
We already have command arg parsing into typed classes. Look at `CommandLineOptions` and `CommandLineArgParser`. 

On Startup, you'll probably need to call:

```
var args = new CommandLineArgParser().Parse();
if(args.SilentStart) ...
```

Reply:
I saw these classes but decided that they are not currently useful here. And looking deeply today I have the same feelings. E.g. because
`new CommandLineArgParser().Parse();`
implemented to work only with URLs params. 
And I don't see any reason to use `EnvironmentWrapper` that uses `Environment.GetCommandLineArgs()` because we already have an access to startup args via `OnStartup` signature.

Let's talk about this tomorrow during our sync?